### PR TITLE
RTF-1397 - Create new version of User strategies

### PR DESCRIPTION
**Pull request by jonatanjakoniuk**
Complete the following tasks before merging in your PR:
- [x] If this is a user facing change, update [version.txt](https://github.com/relativitydev/relativity.testing.framework.api/blob/master/version.txt) and [CHANGELOG.md](https://github.com/relativitydev/relativity.testing.framework.api/blob/master/CHANGELOG.md)

### CODE REVIEW

File: source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/Querying/QueryStrategy`.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+﻿using System;
```
Comment (briantoms) [inline source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/Querying/QueryStrategy`.cs:1]: File should be named ``QueryStrategy`1.cs``, not ``QueryStrategy`.cs`` if I understand correctly.

Comment (briantoms) [reply source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/Querying/QueryStrategy`.cs:1]: Also, why is there ResourcePools stuff in this PR? Seems unrelated.

**Review by briantoms — APPROVED** [review]

File: source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/Querying/QueryStrategy`.cs (hunk 1 above)
Comment (paulletang) [reply source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/Querying/QueryStrategy`.cs:1]: `QueryStrategy  is used by `UserGetGroupsStrategy` as well as the ResourcePool stuff, so I think they both needed to be refactored.

**Review by paulletang — APPROVED** [review]

File: source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/Querying/QueryStrategy`.cs (hunk 1 above)
Comment (briantoms) [reply source/Relativity.Testing.Framework.Api/Strategies/ResourcePools/Querying/QueryStrategy`.cs:1]: If it's a shared base class, it shouldn't be in the `ResourcePools` folder.

**Review by paulletang — APPROVED** [review]

**Pull request merged**